### ISSUE TITLE
indexedMap doesn't need to return a Maybe

### ISSUE THREAD

**Issue by ccapndave** [tomjkidd/elm-multiway-tree-zipper#14]
Since `mapListOverTree` only returns `Nothing` if its list is empty, `indexedMap` builds a list based on the length of the tree, and a multiway tree always has at least one node I don't think there is any way for `indexedMap` to actually return `Nothing`.

**Comment by tomjkidd** [comment]
Yeah, your analysis seems correct. I'll look into cleaning it up. Thanks.
